browser-test/src/support/applicant_questions.ts

```diff
@@ -214,10 +214,14 @@ export class ApplicantQuestions {
   }
 
   /** On the review page, click "Edit" to change an answer to a previously answered question. */
-  async editQuestionFromReviewPage(questionText: string) {
-    await this.page.click(
-      `.cf-applicant-summary-row:has(div:has-text("${questionText}")) a:has-text("Edit")`,
-    )
+  async editQuestionFromReviewPage(
+    questionText: string,
+    northStarEnabled = false,
+  ) {
+    const element = northStarEnabled
+      ? `.block-summary:has(div:has-text("${questionText}")) a:has-text("Edit")`
```

Comment:
**suggestion (non-blocking):** Would you mind using a locator?  I think that would have the advantage of making use of Playwright's auto-waiting and retrying abilities.

Reply:
Updated!